AWS-CodePipeline/sam-monorepo-pipeline-template/questions.json

```
@@ -0,0 +1,234 @@
+            "question": "What is the sam application stack name for stage 2?",
+            "isRequired": true,
+            "default": "sam-app-prod"
+        },
+        {
+            "key": "prod_pipeline_execution_role",
+            "question": "What is the pipeline execution role ARN for stage 2?",
+            "isRequired": true,
+            "allowAutofill": true,
+            "default": {
+                "keyPath": [
+                    {
+                        "valueOf": "prod_stage_name"
+                    },
+                    "pipeline_execution_role"
+                ]
+            }
+        },
+        {
+            "key": "prod_cloudformation_execution_role",
+            "question": "What is the CloudFormation execution role ARN for stage 2?",
+            "isRequired": true,
+            "allowAutofill": true,
+            "default": {
+                "keyPath": [
+                    {
+                        "valueOf": "prod_stage_name"
+                    },
+                    "cloudformation_execution_role"
+                ]
+            }
+        },
+        {
+            "key": "prod_artifacts_bucket",
+            "question": "What is the S3 bucket name for artifacts for stage 2?",
+            "isRequired": true,
+            "allowAutofill": true,
+            "default": {
+                "keyPath": [
+                    {
+                        "valueOf": "prod_stage_name"
+                    },
+                    "artifacts_bucket"
+                ]
+            }
+        },
+        {
+            "key": "prod_image_repository",
+            "question": "What is the ECR repository URI for stage 2?",
+            "allowAutofill": true,
+            "default": {
+                "keyPath": [
+                    {
+                        "valueOf": "prod_stage_name"
+                    },
+                    "image_repository"
+                ]
+            }
+        },
+        {
+            "key": "prod_region",
+            "question": "What is the AWS region for stage 2?",
+            "isRequired": true,
+            "allowAutofill": true,
+            "default": {
+                "keyPath": [
+                    {
+                        "valueOf": "prod_stage_name"
+                    },
+                    "region"
+                ]
+            }
+        },
+        {
+            "key": "message_prod_stage_configured",
+            "question": "Stage 2 configured successfully.\n",
+            "kind": "info"
+        },
+        {
+            "key": "pipeline_stack_name",
+            "question": "What is the CloudFormation stack name for this Pipeline?",
+            "isRequired": true
+        },
+        {
+            "key": "instructions",
+            "question": "To deploy this template and connect to the main git branch, run this against the leading account:\n`sam deploy --config-file pipeline/samconfig-pipeline.toml --config-env pipeline`.",
```

This will show up as:

```text
To deploy this template and connect to the main git branch, run this against the leading account:
`sam deploy --config-file pipeline/samconfig-pipeline.toml --config-env pipeline`.
```

The final period is a little awkward on its own line. Perhaps:

```suggestion
            "question": "To deploy this template and connect to the main Git branch, run this against the leading account: `sam deploy --config-file pipeline/samconfig-pipeline.toml --config-env pipeline`.",
```

or (so that the line can be copy-pasted with a triple-click):

```suggestion
            "question": "To deploy this template and connect to the main Git branch, run this against the leading account:\nsam deploy --config-file pipeline/samconfig-pipeline.toml --config-env pipeline",
```